Rubberduck.Core/app.config

```diff
@@ -406,8 +406,15 @@
                 InspectionType="RubberduckOpportunities" />
               <CodeInspection Name="OnLocalErrorInspection" Severity="Suggestion"
                 InspectionType="LanguageOpportunities" />
+<<<<<<< HEAD
+              <CodeInspection Name="IsMissingOnInappropriateArgumentInspection"
+                Severity="Warning" InspectionType="CodeQualityIssues" />
+              <CodeInspection Name="IsMissingWithNonArgumentParameterInspection"
+                Severity="Warning" InspectionType="CodeQualityIssues" />
+=======
               <CodeInspection Name="AssignmentNotUsedInspection" Severity="Suggestion"
                 InspectionType="CodeQualityIssues" />
+>>>>>>> next
```

Comment:
merge artifacts?

Reply:
Grrr... HTH did it build with that there? Fixed.